client/state/action-types.ts

```diff
@@ -66,6 +66,15 @@ export const AUTOMATED_TRANSFER_STATUS_REQUEST_ONCE = 'AUTOMATED_TRANSFER_STATUS
 export const AUTOMATED_TRANSFER_STATUS_REQUEST_FAILURE =
 	'AUTOMATED_TRANSFER_STATUS_REQUEST_FAILURE';
 export const AUTOMATED_TRANSFER_STATUS_SET = 'AUTOMATED_TRANSFER_STATUS_SET';
+export const ATOMIC_PLUGIN_INSTALL_INITIATE_WITH_TRANSFER =
+	'ATOMIC_PLUGIN_INSTALL_INITIATE_WITH_TRANSFER';
+export const ATOMIC_PLUGIN_INSTALL_INITIATE = 'ATOMIC_PLUGIN_INSTALL_INITIATE';
+export const ATOMIC_PLUGIN_INSTALL_REQUEST_STATUS = 'ATOMIC_PLUGIN_INSTALL_REQUEST_STATUS';
+export const ATOMIC_PLUGIN_INSTALL_REQUEST_TRANSFER_STATUS =
+	'ATOMIC_PLUGIN_INSTALL_REQUEST_TRANSFER_STATUS';
+export const ATOMIC_PLUGIN_INSTALL_SET_TRANSFER_STATUS =
+	'ATOMIC_PLUGIN_INSTALL_SET_TRANSFER_STATUS';
+export const ATOMIC_PLUGIN_INSTALL_SET_STATUS = 'ATOMIC_PLUGIN_INSTALL_SET_STATUS';
```

Comment:
I think these could use some renaming but I'm not sure what makes sense yet.

Maybe:
```
ATOMIC_SOFTWARE_INITIATE_INSTALL
ATOMIC_SOFTWARE_REQUEST_STATUS
ATOMIC_SOFTWARE_SET_STATUS
```
and
```
ATOMIC_TRANSFER_INITIATE_TRANSFER
ATOMIC_TRANSFER_REQUEST_LATEST
ATOMIC_TRANSFER_SET_LATEST
```

Reply:
I've updated the transfer stuff to talk about "latest transfer" instead of "status" now but software "status" is a bit murky because its the whole response, maybe we just change that to a "applied" string or an error?